test/units/module_utils/common/text/test_converters.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2019, Andrew Klychkov @Andersson007 <aaklychkov@mail.ru>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import json
+import pytest
+
+from datetime import datetime
+from pytz import timezone as tz
+
+from ansible.module_utils.common._collections_compat import Set
+from ansible.module_utils.common.text.converters import (
+    _json_encode_fallback,
+    container_to_bytes,
+    container_to_text,
+    jsonify,
+)
+
+
+DEFAULT_ENCODING = 'utf-8'
+
+
+class Test_json_encode_fallback:
```

Comment:
It Python, you should use a proper camel case for classes:
```suggestion
class TestJsonEncodeFallback:
```